includes/Media.php

```diff
@@ -207,7 +207,7 @@ public static function rest_api_init() {
 	 * @param array    $response   Array of prepared attachment data.
 	 * @param \WP_Post $attachment Attachment object.
 	 *
-	 * @return array<string> $response;
+	 * @return array $response;
```

Comment:
response may be `array<int, mixed>`

Reply:
I disabled that check in PHPStan config though

Thus reverting this change here.

Reply:
Be aware that any array is more like a container than a variable type: not possible to track what kind of values are in it.